src/Microsoft.DotNet.CodeAnalysis/PackageFiles/Microsoft.DotNet.CodeAnalysis_DisablePinvokeAnalyzer.ruleset

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Common diagnostic rules" Description="Enables/disable rules for all projects." ToolsVersion="14.0">
+  <Include Path="$(MSBuildThisFileDirectory)\Microsoft.DotNet.CodeAnalysis.ruleset" Action="Default" />
+  <Rules AnalyzerId="Microsoft.DotNet.CodeAnalysis" RuleNamespace="Microsoft.DotNet.CodeAnalysis.Analyzers">
+    <!-- PinvokeChecker should only be enabled for AnyOS or Windows_NT configurations -->
+    <Rule Id="BCL0100" Action="None" />
```

Comment:
So there will be a ruleset inside CoreFx that will enable this analyzer and we are going to use the ruleset depending on the OS?